### ISSUE TITLE
Implement auth 

### ISSUE THREAD

**Issue by katiemthom** [mbyczkowski/lacocina#163]

**Comment by katiemthom** [comment]
Actually, it looks like clearance provides auth. What's the plan for admin? We can manually create accounts and give them admin privilege? Looks like we need a way to specify admin role in the users table then?

**Comment by mbyczkowski** [comment]
Yeah, I think we just need to have something like `is_admin` field and make sure we check for admin user before we let people into the `/admin` zone.
